### PR TITLE
Check for Subsequence in Python

### PR DESCRIPTION
**Pull request by yashaswibiyahut**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

**YES**

### Description

**Issue no: #993**
Check for subsequence in Python.
Time Complexity : O(n)

_Below is the Output of this code._

### Checklist

- [x]  I've read the contribution guidelines.
- [x]  I've checked the issue list before deciding what to submit.
- [x]  I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

**Comment by yashaswibiyahut** [comment]
WhatsApp Image 2020-09-15 at 4 39 25 PM
Output of above code.

**Comment by yashaswibiyahut** [comment]
kindly review this PR @Kajol-Kumari @SKAUL05 @siddharth25pandey @iamrajiv @HarshCasper

**Review by SKAUL05 — CHANGES_REQUESTED** [review]
Change alphabets to meaningful variable names

**Comment by yashaswibiyahut** [comment]
Done @SKAUL05 
Kindly review it.

**Review by atarax665 — CHANGES_REQUESTED** [review]
Format your code according to [PEP 8](https://www.python.org/dev/peps/pep-0008/).

**Comment by yashaswibiyahut** [comment]
already formatted @atarax665

File: Python/other/Subsequence_check.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Take input for class TestCase
+print("Enter Number of TestCase :")
+testcase= int(input())
```
Comment (iamrajiv) [inline Python/other/Subsequence_check.py]: @yashaswibiyahut Update this to `testcase = int(input())`

**Review by iamrajiv — CHANGES_REQUESTED** [review]

File: Python/other/Subsequence_check.py (hunk 1 above)
Comment (yashaswibiyahut) [reply Python/other/Subsequence_check.py]: done @iamrajiv 
kindly review it.

**Review by SKAUL05 — APPROVED** [review]

**Review by iamrajiv — APPROVED** [review]

**Pull request merged**